plugins/linux-capture/xcomposite-input.c

```diff
@@ -615,7 +623,7 @@ static void xcompcap_video_tick(void *data, float seconds)
 
 	// Reacquire window after interval or immediately if reconfigured.
 	s->window_check_time += seconds;
-	bool window_lost = !xcomp_window_exists(conn, s->win);
+	bool window_lost = !xcomp_window_exists(conn, s->win) || !s->gltex;
```

Comment:
Looks correct, thanks! Could you please separate this change into another commit?